akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/HttpChallenge.scala

```diff
@@ -13,7 +13,8 @@ final case class HttpChallenge(scheme: String, realm: String,
                                params: Map[String, String] = Map.empty) extends jm.headers.HttpChallenge with ValueRenderable {
```

Comment:
@ianclegg Could you please add the additional constructor that @ktoso [mentioned](https://github.com/akka/akka/issues/20786#issuecomment-227004833)?

We could use that one here [instead](https://github.com/akka/akka/blob/c66ce62d633885d73f6d7603df34303bd8ba9993/akka-http-core/src/main/scala/akka/http/impl/model/parser/CommonRules.scala#L177-L177).